src/batch/src/executor/join/lookup_join.rs

```diff
@@ -315,20 +318,46 @@ impl<P: 'static + ProbeSideSourceBuilder> Executor for LookupJoinExecutor<P> {
 }
 
 impl<P: 'static + ProbeSideSourceBuilder> LookupJoinExecutor<P> {
+    #[try_stream(boxed, ok = Vec<DataChunk>, error = RwError)]
```

Comment:
maybe make this a util function in `common/src/array/datachunk`?

Reply:
Ok